RunSim.py

```diff
@@ -42,29 +42,32 @@ def allSimSettings(config):
 
 def main(configfile, cleps, noui):
 
-    # log start of simulation
+    # log
     log.info(f'RunSim starting ...')
 
-    config = toml.load(pkg_resources.resource_filename(__name__, f"configs/{configfile}.toml"))
-    config = config['atlas']
+    # create a list of simSettings, one per simulation run
+
```

Comment:
remove line between paragraph comment and code. Newline only after separator (`#======`)